### PR TITLE
Added resume feature in fuzzy finder

### PR DESCRIPTION
**Pull request by Mariappan**
Added support for fuzzy finder resume.

In the existing code, palette has been used for both finder and command line.
But since we need to resume the fuzzy finder operation, created a separate palette for fizzy finder alone to simplify the resume process.

### CODE REVIEW

**Comment by Mariappan** [comment]
I am getting crashes after this two checkins in linux

* 8a38b53 - (upstream/multigrid, origin/multigrid) Improve startup performance (25 hours ago) <akiyosi>
* dc809db - Improve performance degradation (2 days ago) <akiyosi>

**Comment by Mariappan** [comment]
Crashes with 
goroutine 23 [running]:
github.com/Mariappan/gonvim/editor.(*Palette).resize(0x0)
	/home/maari/go/src/github.com/Mariappan/gonvim/editor/palette.go:196 +0x60
github.com/Mariappan/gonvim/editor.(*Workspace).updateSize(0xc00049ea80)
	/home/maari/go/src/github.com/Mariappan/gonvim/editor/workspace.go:749 +0x11e
github.com/Mariappan/gonvim/editor.(*Tabline).subscribe(0xc000126000)
	/home/maari/go/src/github.com/Mariappan/gonvim/editor/tabline.go:54 +0x6d
github.com/Mariappan/gonvim/editor.(*Workspace).attachUI(0xc00049ea80, 0x0, 0x0, 0x0, 0x0)
	/home/maari/go/src/github.com/Mariappan/gonvim/editor/workspace.go:340 +0x12f
github.com/Mariappan/gonvim/editor.(*Workspace).init(0xc00049ea80, 0x0, 0x0)
	/home/maari/go/src/github.com/Mariappan/gonvim/editor/workspace.go:305 +0x4d
created by github.com/Mariappan/gonvim/editor.(*Workspace).startNvim
	/home/maari/go/src/github.com/Mariappan/gonvim/editor/workspace.go:294 +0x27f

Calling startNvim before palette init causes this. We may need to move startNvim after initing all GUI elements.

**Comment by Mariappan** [comment]
Moving the w.startNvim() causes this crash

The following diff worked for me:

``` diff
diff --git a/editor/workspace.go b/editor/workspace.go
index 555cae2..7c956d9 100644
--- a/editor/workspace.go
+++ b/editor/workspace.go
@@ -112,8 +112,6 @@ func newWorkspace(path string) (*Workspace, error) {
        w.message = initMessage()
        w.message.ws = w
 
-       go w.startNvim(path)
-
        w.registerSignal()
 
        w.screen = newScreen()
@@ -181,6 +179,8 @@ func newWorkspace(path string) (*Workspace, error) {
 
        go w.minimap.startMinimapProc()
 
+       go w.startNvim(path)
+
        if runtime.GOOS == "windows" {
                select {
                case <-w.doneNvimStart:
```

File: editor/message.go [hunk 1]
```diff
@@ -256,6 +256,9 @@ func (m *Message) msgShow(args []interface{}) {
 			replaceLast, ok = arg.([]interface{})[2].(bool)
 		}
 
+		if kind == "" && text == "<" {
+			continue
+		}
```
[the remark this answers is not in the document]
Comment (akiyosi) [reply editor/message.go]: I know that.
This is probably a neovim bug, and I am not aggressive in dealing with the GUI.

**Comment by akiyosi** [comment]
It is looks good to me.
I will merge this PR.

**Pull request merged**